via/views/__init__.py

```diff
@@ -20,6 +20,12 @@ def add_routes(config):  # pragma: no cover
         "/google_drive/{file_id}/{resource_key}/proxied.pdf",
         factory=QueryURLResource,
     )
+    config.add_route(
+        "proxy_one_drive_pdf",
+        "/one_drive/proxied.pdf",
```

Comment:
It's actually "OneDrive" (all one word) :)

```suggestion
        "proxy_onedrive_pdf",
        "/onedrive/proxied.pdf",
```

I'd recommend this throughout, e.g. `_MS_ONEDRIVE_URL` etc.

Also because there are actually three logical parts to `"proxy_foo_pdf"` not four, so even if the `foo` in the middle really was two separate words you'd run them together. This is how `_`'s are used in names in the standard library and most big third-party libraries: `bytecode_helper` (not `byte_code_helper`), `itertools` not `iter_tools`, etc. Basically you insert the `_`'s when they're necessary for readability, not between every word

Reply:
Updated to "ONEDRIVE" all around.